library/src/main/java/com/google/android/exoplayer/extractor/flv/AudioTagPayloadReader.java

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright (C) 2014 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.android.exoplayer.extractor.flv;
+
+import android.util.Log;
+import android.util.Pair;
+
+import com.google.android.exoplayer.C;
+import com.google.android.exoplayer.MediaFormat;
+import com.google.android.exoplayer.extractor.TrackOutput;
+import com.google.android.exoplayer.util.CodecSpecificDataUtil;
+import com.google.android.exoplayer.util.MimeTypes;
+import com.google.android.exoplayer.util.ParsableBitArray;
+import com.google.android.exoplayer.util.ParsableByteArray;
+
+import java.util.Collections;
+
+/**
+ * Parses audio tags of from an FLV stream and extracts AAC frames.
+ */
+final class AudioTagPayloadReader extends TagPayloadReader {
+
+  private static final String TAG = "AudioTagPayloadReader";
+
+  // Sound format
+  private static final int AUDIO_FORMAT_LINEAR_PCM_PLATFORM_ENDIAN = 0;
```

Comment:
A whole bunch of these aren't used anywhere (applies to TAG above also). Can we delete all of the unused ones? Not sure what IDE you're using, but I always turn on a warning that highlights unused stuff, which makes them easy to spot and delete.